hlib/python/hlib/frontend/test/gen_imagenet.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import print_function
+#import keras
+#from keras.datasets import cifar100
+#from keras.preprocessing.image import ImageDataGenerator
+#from keras.models import Model, Sequential
+#from keras.layers import Dense, Dropout, Activation, Flatten, Input
+#from keras.layers import Conv2D, MaxPooling2D
+import numpy as np
+import random
+import cv2
+import matplotlib.pyplot as plot
+import os
+import tarfile
+import torchvision
+root = "/home/pbc48/install/datasets/imagenet_2012/images/val"
```

Comment:
We should remove this. You should add this to our internal documentation, like your report. Please send me an updated report after that.

Reply:
This is used to demo model accuracy, not exactly needed for testing

Reply:
Ideally, we want to write something like this.
```python
# set the path to your data set
# root = ".../imagnet_2012/images/val"
```

Reply:
And within your report, you can specify what are the "..."

Reply:
We could write something like this.
```python
# set your path to the data set
# root = ".../imagenet_2012/images/val"
```
And in your report, specify what exactly is "..."

Reply:
Sure thing